uint32 numberOfSegments
---
float32[] responseProbabilities
bool success